config/checkstyle_non_main_files_checks.xml

```diff
@@ -24,6 +24,15 @@
     <message key="regexp.filename.mismatch"
       value="CI folder should only contain executables files"/>
   </module>
+  <module name="RegexpOnFilename">
+    <property name="id" value="regexpOnScriptsAndConfigs"/>
+    <property name="folderPattern" value="[\\/].github[\\/]workflows([\\/]|$)"/>
+    <property name="fileNamePattern" value="^([a-z][a-z0-9]*)(-[a-z0-9]+)*\..*$"/>
+    <property name="match" value="false"/>
+    <property name="fileExtensions" value="sh, groovy, cmd, bat, yml"/>
+    <message key="regexp.filename.mismatch"
+             value="Filename should be kebab-case."/>
```

Comment:
`Filename should be in kebab-case.`

>then the file name should be in PascalCase.

https://docs.devland.is/technical-overview/adr/0009-naming-files-and-directories

Reply:
Alright. Done. I wish Grammarly was avaialbe for IntelliJ Idea so it could spot this :D

Reply:
Still not done

Reply:
Should be done now. I thought I did the change back then, huh.